inc/class-book.php

```diff
@@ -490,6 +494,9 @@ static function deleteBookObjectCache() {
 		static::$preview = [];
 		static::$__order = [];
 
+		// Subsections
+		delete_transient( \Pressbooks\Book::SUBSECTIONS_TRANSIENT );
```

Comment:
I prefer `delete_transient( static::SUBSECTIONS_TRANSIENT );` because it's referencing itself here.

Reply:
```suggestion
		delete_transient( static::SUBSECTIONS_TRANSIENT );
```